cv_next/app/cv/[cvId]/components/commentSection/comment.tsx

```diff
@@ -28,69 +29,71 @@ export default function Comment({
   commentsOfComment = [],
   setCommentsOfComments,
 }: {
-  comment: CommentModel
-  userId: string
-  childDepth?: number
-  cv: CvModel
-  commentsOfComment: any[]
+  comment: CommentModel;
+  userId: string;
+  childDepth?: number;
+  cv: CvModel;
+  commentsOfComment: any[];
   setCommentsOfComments: (
     update: (prev: Map<string, any[]>) => Map<string, any[]>
-  ) => void
+  ) => void;
 }) {
   const [commentOnCommentStatus, setCommentOnCommentStatus] =
-    useState<boolean>(false)
-  const [commentOnComment, setCommentOnComment] = useState<string>("")
-  const [submitCommentOnComment, setSubmitCommentOnComment] =
-    useState<boolean>(false)
-
-  const [showAlert, setShowAlert] = useState<boolean>(false)
+    useState<boolean>(false);
+  const [commentOnComment, setCommentOnComment] = useState<string>("");
+  const router = useRouter();
+  const [showAlert, setShowAlert] = useState<boolean>(false);
   const onAlertClick = async (type: boolean) => {
-    if (type) await deleteCommentAction()
-    setShowAlert(false)
-  }
+    if (type) await deleteCommentAction();
+    setShowAlert(false);
+  };
+  const supabase = useSupabase();
 
   const addNewCommentClickEvent = async () => {
-    if (!userId) throw new Error("User not found")
-    const commentToAdd: NewCommentModel = {
-      data: commentOnComment,
-      document_id: cv.id,
-      parent_comment_Id: comment.id,
-      user_id: userId,
-    }
+    const userId = (await supabase.auth.getUser()).data.user?.id;
+    if (!userId) router.push("/inactive");
```

Comment:
Perhaps to `/login` instead?
Also good practice would be to redirect with a return url so that the user would authenticate then come back to the same page, back we can handle this later.

Reply:
The login page in this version doesn't have the next param(it's with the SSR), but will add it as a TODO

Reply:
I uploaded a fix for that in the SSR branch as this one doesn;t have the next page param